docs/data-sources/kafka_cluster_v2_instance.md

```diff
@@ -45,7 +45,7 @@ The following terms are used to describe attributes in the schema of this data s
 <br>Adds the specified version of Kafka Karapace REST Proxy to this Kafka cluster.<br><br>
 *___kafka_version___*<br>
 <ins>Type</ins>: string, read-only<br>
-<ins>Constraints</ins>: pattern: `[0-9]+\.[0-9]+\.[0-9]+`<br><br>Version of Kafka to run on the cluster. Available versions: <ul> <li>`3.0.2`</li> <li>`3.1.2`</li> <li>`2.8.2`</li> <li>`2.7.1`</li> </ul><br><br>
```

Comment:
We'll need a patch version bump in the makefile for these changes to kick in on the terraform registry docs

Reply:
Thanks, missed that one. Added it now.